polymer-punk-brew/src/punk-brew-app/punk-brew-app.html

```diff
@@ -42,7 +42,7 @@
     </iron-ajax>
 
 
-    <app-header effects="waterfall">
```

Comment:
Weirdly, this effect worked even though it would error saying it was undefined 🤔